### ISSUE TITLE
Package ‘spring-103.0’ broken

### ISSUE THREAD

**Issue by Dr-Lambda** [NixOS/nixpkgs#39689]
## Issue description

Spring package is broken.

This is the error:
```
building Nix...
building the system configuration...
[31;1merror:[0m Package ‘spring-103.0’ in /nix/store/xdfsm3pdjalxl4l8lr7dswidhh8yr0qp-nixos-18.03.132021.c0c5571ec1a/nixos/pkgs/games/spring/default.nix:49 is marked as broken, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.

(use '--show-trace' to show detailed location information)
```

### Steps to reproduce

Try to install spring or springLobby from nixpkgs.

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.34, NixOS, 18.03.132021.c0c5571ec1a (Impala)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.0`
 - channels(lambda): `"nixos-17.03.1775.56da88a298"`
 - channels(root): `"nixos-18.03.132021.c0c5571ec1a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by lheckemann** [comment]
This is because it's been found not to work in the past and no maintainer has stepped up to fix it. It's been retained for anyone who may want to fix it. Feel free to contribute, it's unlikely to get fixed otherwise since it's been broken since September (40ed22650737a444fee7883e69e852fd4b808133).

cc @phreedom @qknight @domenkozar (package's listed maintainers)

**Comment by JohnAZoidberg** [comment]
Seems like it's been fixed with #41153 :)
The game also starts successfully for me.

**Comment by lheckemann** [comment]
Thanks for the triage help @JohnAZoidberg !
